calico/felix/config.py

```diff
@@ -212,6 +212,9 @@ def __init__(self, config_path):
         self.add_parameter("IpInIpMtu",
                            "MTU to set on the IP-in-IP device", 1440,
                            value_is_int=True)
+        self.add_parameter("IpInIpTunnelAddr",
+                           "IPv4 address to set on the IP-in-IP device",
+                           "none")
```

Comment:
This only supports adding IPv4 addresses (but the name isn't clear about that).

What's the plan for extending this feature to support IPv6 if needed?  Do we remove the "IPv4" restriction but keep this field (what happens with dual stack?)?  If not, and we're going to add a second field, I'd like to see this one named for IPv4 from the start so we don't need to change it.